### PR TITLE
Adding IS entry for key manager in vm

### PR DESCRIPTION
**Pull request by pubudu538**
- Added a new vm entry for IS as a key manager
- Changed versions in kubernetes samples
- Added folders for vms

### CODE REVIEW

File: wso2am/1.9.1/samples/applications/wso2am-191-application-with-identity-server/artifacts/wso2am-191-application.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
                         }
                     },
                     {
-                        "type": "wso2is-500-manager",
+                        "type": "wso2is-km-500-manager",
```
Comment (ravihansa3000) [inline wso2am/1.9.1/samples/applications/wso2am-191-application-with-identity-server/artifacts/wso2am-191-application.json]: shall we put wso2is-as-km?

File: common/vm/puppet/manifests/nodes/nodes.pp [hunk 2]
```diff
@@ -37,8 +37,8 @@
   class { 'python_agent': }
   class { 'configurator': }
   class { 'wso2installer':
-    server_name      => 'wso2is5.0.0',
-    module_name      => 'wso2is-500'
+    server_name      => 'wso2is-5.0.0',
+    module_name      => 'wso2is500'
```
Comment (ravihansa3000) [inline common/vm/puppet/manifests/nodes/nodes.pp]: why is it named "module_name" ?

File: wso2am/1.9.1/samples/applications/wso2am-191-application-with-identity-server/artifacts/wso2am-191-application.json (hunk 1 above)
Comment (pubudu538) [inline wso2am/1.9.1/samples/applications/wso2am-191-application-with-identity-server/artifacts/wso2am-191-application.json]: Changed to is-as-km

File: common/vm/puppet/manifests/nodes/nodes.pp (hunk 2 above)
Comment (pubudu538) [inline common/vm/puppet/manifests/nodes/nodes.pp]: This is pack folder name used in all the puppet modules

**Pull request merged**